app/models/user.rb

```diff
@@ -2,9 +2,8 @@
 
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+  # :confirmable, :lockable, :omniauthable, :database_authenticatable, :registerable, :recoverable, :validatable
+  devise :saml_authenticatable, :trackable, :rememberable, :timeoutable
```

Comment:
Are :rememberable, :timeoutable still needed w/ saml auth?

Reply:
I think we need timeoutable, but not rememberable.